README.md

```diff
@@ -1 +1,23 @@
-# parasoft-demo-app-mobile
\ No newline at end of file
+# parasoft-demo-app-mobile
+The Parasoft Demo Application is an example Android application. The application is configurable, and is used to demonstrate functionality in a variety of Parasoft tools.
+
+## Getting Started
+### Create emulator
+1. Open **Device Manager**, click **Create device**.
+2. Choose a device which has the resolution higher than **720x1280**, we recommend the device **Pixel 5 (1080x2340)**.
+3. Click **next**, choose an API version which **higher or equal to 32**, then click finish.
+
+### Build apk
+Select the device that you just created, and click **Run 'app'**, Android stdio will build and open the app automatically.
+
+## Using the Demo Application
+1. Make sure the PDA service is running
+2. Click the setting icon in the top right-hand corner, enter the Base URL. The default URL is **http://10.0.2.2.8080** which is the base URL of the local PDA service.
+3. Enter the username and password to sign in. The default username and password are:
+
+| Option   | value    |
```

Comment:
Change to something like:
| Username | Password |
|------------|------------|
| approver  | password |